shared/components/StudyInfo/Details.tsx

```diff
@@ -73,9 +74,14 @@ const CollapsedDetailsTitle = styled("strong")`
   margin: 10px 0 8px 0;
 `
 
-const AiemmanOsaamisenTitleExpanded = styled("strong")`
+const ExpandedDetailsTitle = styled("strong")`
   display: block;
-  margin: 10px 0 8px 20px;
+  margin: 10px 0 8px;
+  margin-left: ${props => props.theme.spacing.l};
```

Comment:
Example for that https://github.com/Opetushallitus/ehoks-ui/blob/master/shared/components/StudyInfo/OsaamisenOsoittaminen.tsx#L40

Reply:
Changed